examples/hello/lib/hello/app.rb

```diff
@@ -0,0 +1,66 @@
+require "hello/actions"
+
+module Hello
+  class App
+    class Router
```

Comment:
Then I can start moving my example app's request specs into wren router unit tests.

I will also want to start driving some `[resource_name]_path` methods with unit tests.